README.md

```diff
@@ -18,6 +20,10 @@ It requires VIM >= 8.0 and VIM compiled with `channel`, `signs` and `job` suppor
 8. use the command `:JDBStepOver` to execute to the next line
 9. with `:JDBCommand` you can send any JDB command to the JDB JAVA process, e.g. you want to see all locals do `:JDBCommands locals`
 10. with `:JDBContinue` you can resume the execution until the next breakpoint is hits
+11. with `:JDBDebugProcess` will (currently) pick the function name you are currently in if it is annontated with "@Test". It will then construct the the path to the class from the class name and package name in the file.
```

Comment:
The command name is misleading in my opinion. I would prefer JDBDebugTest or JDBDebugMethod